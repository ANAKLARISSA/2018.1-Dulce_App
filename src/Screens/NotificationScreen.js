import React from 'react';
import {Text} from 'react-native';

class NotificationScreen extends React.Component {

  constructor(props) {
    super(props);
    this.state = {
      data: {}
    };
  }

  render() {
    return (
      <Text> Esta tela é a de notificações</Text>
    );
  }
}

export default NotificationScreen;
