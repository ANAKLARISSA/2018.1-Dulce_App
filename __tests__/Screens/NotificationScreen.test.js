import React from 'react';
import 'react-native';
import NotificationScreen from '../../src/Screens/NotificationScreen';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import {shallow} from 'enzyme';

require('bezier');
Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer
  .create(<NotificationScreen />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

/* it('onPress Aceitar',() => {

  const item = {
    name: 'ezequiel'
  }
const onPressMock = jest.fn();
const _root = jest.fn();
const wrapper = shallow(<NotificationScreen onPress={onPressMock}/>);
const List = wrapper.find('Styled(List)').at(0);
const renderR = shallow(List.props().renderRow(item));
const refused = renderR.find('ValidateButton').at(0);
refused.props().onPress();
}); */
