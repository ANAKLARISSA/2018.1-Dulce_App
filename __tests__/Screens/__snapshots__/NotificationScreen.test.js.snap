// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fd7d01",
        "flex": 0.25,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#f1f1f1",
          "borderRadius": 5,
          "height": 40,
          "margin": 1,
          "width": 7.5,
        }
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "height": 1334,
        },
        Object {
          "flex": 8,
        },
      ]
    }
    virtual={undefined}
  >
    <View
      onLayout={[Function]}
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#F8F8F8",
              "borderBottomColor": "#a7a6ab",
              "borderBottomWidth": 0.5,
              "elevation": 3,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": 18,
              "right": 0,
              "shadowColor": undefined,
              "shadowOffset": undefined,
              "shadowOpacity": undefined,
              "shadowRadius": undefined,
              "top": 0,
            },
            Object {
              "backgroundColor": "#5f4b8b",
            },
          ]
        }
        virtual={undefined}
      >
        <View />
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flex": 1,
              },
              Object {
                "alignItems": "center",
              },
            ]
          }
          virtual={undefined}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 17,
                  "fontWeight": "700",
                  "marginLeft": undefined,
                  "paddingLeft": 4,
                  "paddingTop": 1,
                  "textAlign": "center",
                },
                Object {
                  "fontFamily": "Raleway",
                },
              ]
            }
            virtual={undefined}
          >
             
            Notificações
             
          </Text>
        </View>
      </View>
    </View>
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": undefined,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      viewIsInsideTabBar={false}
      virtual={undefined}
    >
      <View>
        <RCTScrollView
          closeOnRowBeginSwipe={false}
          closeOnRowPress={true}
          closeOnScroll={true}
          dataArray={
            Array [
              Object {
                "currentDay": "2018-04-28",
                "currentEndTime": "12:00",
                "currentStartTime": "10:30",
                "name": "Guilherme",
                "requestDay": "2018-05-12",
                "requestEndTime": "12:00",
                "requestStartTime": "10:30",
              },
              Object {
                "currentDay": "2018-04-28",
                "currentEndTime": "12:00",
                "currentStartTime": "10:30",
                "name": "Ezequiel",
                "requestDay": "2018-03-30",
                "requestEndTime": "12:00",
                "requestStartTime": "10:30",
              },
            ]
          }
          dataSource={
            ListViewDataSource {
              "items": 2,
            }
          }
          directionalDistanceChangeThreshold={2}
          disableLeftSwipe={false}
          disableRightSwipe={false}
          enableEmptySections={true}
          leftOpenValue={0}
          previewFirstRow={false}
          recalculateHiddenLayout={false}
          renderRow={[Function]}
          renderScrollComponent={[Function]}
          rightOpenValue={0}
          style={Object {}}
          swipeToOpenPercent={50}
          virtual={undefined}
        >
          <View>
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#ccc",
                  "borderLeftWidth": 0.5,
                  "borderRadius": 2,
                  "borderRightWidth": 0.5,
                  "borderTopWidth": 0.5,
                  "elevation": 3,
                  "flex": 1,
                  "flexWrap": "nowrap",
                  "marginBottom": 5,
                  "marginLeft": 2,
                  "marginRight": 2,
                  "marginTop": 5,
                  "shadowColor": "#000",
                  "shadowOffset": Object {
                    "height": 2,
                    "width": 0,
                  },
                  "shadowOpacity": 0.1,
                  "shadowRadius": 1.5,
                }
              }
              virtual={undefined}
            >
              <View
                header={true}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderBottomWidth": null,
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 15,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 15,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "#1a1515",
                      "fontFamily": "Lato",
                      "fontSize": 16,
                      "textDecorationStyle": "solid",
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#5f4b8b",
                        "fontFamily": "Raleway",
                        "fontSize": 15,
                      }
                    }
                  >
                    Guilherme
                  </Text>
                   solicitou mudança com o seu horário
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
                virtual={undefined}
              >
                <View
                  style={
                    Object {
                      "alignItems": "flex-start",
                      "alignSelf": "stretch",
                      "flex": 1,
                    }
                  }
                  virtual={undefined}
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                        "flexDirection": "row",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        Seu Horário
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        2018-04-28
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        10:30
                         - 
                        12:00
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={false}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": undefined,
                              "fontSize": 12,
                            },
                            Array [
                              Object {
                                "color": "#000",
                                "fontSize": 30,
                              },
                              Object {
                                "color": "#000",
                                "fontSize": 30,
                              },
                              Object {
                                "color": "#5f4b8b",
                                "fontSize": 40,
                              },
                            ],
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                          ]
                        }
                        virtual={undefined}
                      >
                        
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "flex-end",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        Guilherme
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        2018-05-12
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        10:30
                         - 
                        12:00
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
              <View
                footer={true}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderBottomWidth": null,
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
                virtual={undefined}
              >
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginLeft": "60%",
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "transparent",
                        "borderBottomWidth": null,
                        "borderColor": "#007aff",
                        "borderLeftWidth": null,
                        "borderRadius": 5,
                        "borderRightWidth": null,
                        "borderTopWidth": null,
                        "elevation": 0,
                        "flexDirection": "row",
                        "height": 45,
                        "justifyContent": "space-between",
                        "opacity": 1,
                        "padding": 8,
                        "paddingBottom": 6,
                        "paddingTop": 6,
                        "shadowColor": null,
                        "shadowOffset": null,
                        "shadowOpacity": null,
                        "shadowRadius": null,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#fd7d01",
                          "fontFamily": "Raleway",
                          "fontSize": 15,
                        }
                      }
                    >
                      Recusar
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "transparent",
                        "borderBottomWidth": null,
                        "borderColor": "#007aff",
                        "borderLeftWidth": null,
                        "borderRadius": 5,
                        "borderRightWidth": null,
                        "borderTopWidth": null,
                        "elevation": 0,
                        "flexDirection": "row",
                        "height": 45,
                        "justifyContent": "space-between",
                        "opacity": 1,
                        "padding": 8,
                        "paddingBottom": 6,
                        "paddingTop": 6,
                        "shadowColor": null,
                        "shadowOffset": null,
                        "shadowOpacity": null,
                        "shadowRadius": null,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#5f4b8b",
                          "fontFamily": "Raleway",
                          "fontSize": 15,
                        }
                      }
                    >
                      Aceitar
                    </Text>
                  </View>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#ccc",
                  "borderLeftWidth": 0.5,
                  "borderRadius": 2,
                  "borderRightWidth": 0.5,
                  "borderTopWidth": 0.5,
                  "elevation": 3,
                  "flex": 1,
                  "flexWrap": "nowrap",
                  "marginBottom": 5,
                  "marginLeft": 2,
                  "marginRight": 2,
                  "marginTop": 5,
                  "shadowColor": "#000",
                  "shadowOffset": Object {
                    "height": 2,
                    "width": 0,
                  },
                  "shadowOpacity": 0.1,
                  "shadowRadius": 1.5,
                }
              }
              virtual={undefined}
            >
              <View
                header={true}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderBottomWidth": null,
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 15,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 15,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "#1a1515",
                      "fontFamily": "Lato",
                      "fontSize": 16,
                      "textDecorationStyle": "solid",
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#5f4b8b",
                        "fontFamily": "Raleway",
                        "fontSize": 15,
                      }
                    }
                  >
                    Ezequiel
                  </Text>
                   solicitou mudança com o seu horário
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
                virtual={undefined}
              >
                <View
                  style={
                    Object {
                      "alignItems": "flex-start",
                      "alignSelf": "stretch",
                      "flex": 1,
                    }
                  }
                  virtual={undefined}
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                        "flexDirection": "row",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        Seu Horário
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        2018-04-28
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        10:30
                         - 
                        12:00
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={false}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": undefined,
                              "fontSize": 12,
                            },
                            Array [
                              Object {
                                "color": "#000",
                                "fontSize": 30,
                              },
                              Object {
                                "color": "#000",
                                "fontSize": 30,
                              },
                              Object {
                                "color": "#5f4b8b",
                                "fontSize": 40,
                              },
                            ],
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                          ]
                        }
                        virtual={undefined}
                      >
                        
                      </Text>
                    </View>
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "flex-end",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        Ezequiel
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        2018-03-30
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "#1a1515",
                            "fontFamily": "Lato",
                            "fontSize": 16,
                            "textDecorationStyle": "solid",
                          }
                        }
                      >
                        10:30
                         - 
                        12:00
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
              <View
                footer={true}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderBottomWidth": null,
                    "borderRadius": 2,
                    "flexDirection": "row",
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
                virtual={undefined}
              >
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                      "marginLeft": "60%",
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "transparent",
                        "borderBottomWidth": null,
                        "borderColor": "#007aff",
                        "borderLeftWidth": null,
                        "borderRadius": 5,
                        "borderRightWidth": null,
                        "borderTopWidth": null,
                        "elevation": 0,
                        "flexDirection": "row",
                        "height": 45,
                        "justifyContent": "space-between",
                        "opacity": 1,
                        "padding": 8,
                        "paddingBottom": 6,
                        "paddingTop": 6,
                        "shadowColor": null,
                        "shadowOffset": null,
                        "shadowOpacity": null,
                        "shadowRadius": null,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#fd7d01",
                          "fontFamily": "Raleway",
                          "fontSize": 15,
                        }
                      }
                    >
                      Recusar
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "transparent",
                        "borderBottomWidth": null,
                        "borderColor": "#007aff",
                        "borderLeftWidth": null,
                        "borderRadius": 5,
                        "borderRightWidth": null,
                        "borderTopWidth": null,
                        "elevation": 0,
                        "flexDirection": "row",
                        "height": 45,
                        "justifyContent": "space-between",
                        "opacity": 1,
                        "padding": 8,
                        "paddingBottom": 6,
                        "paddingTop": 6,
                        "shadowColor": null,
                        "shadowOffset": null,
                        "shadowOpacity": null,
                        "shadowRadius": null,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#5f4b8b",
                          "fontFamily": "Raleway",
                          "fontSize": 15,
                        }
                      }
                    >
                      Aceitar
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RCTScrollView>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
